system-x/services/redis/pom.xml

```diff
@@ -0,0 +1,32 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>software.tnb</groupId>
+        <artifactId>system-x-services</artifactId>
+        <version>1.0-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>system-x-redis</artifactId>
+    <version>1.0-SNAPSHOT</version>
+    <name>TNB :: System-X :: Services :: Redis</name>
+
+    <properties>
+        <lettuce.core.version>6.2.4.RELEASE</lettuce.core.version>
```

Comment:
could you please upgrade and test with `6.2.5.RELEASE`, it should solve the netty downgrade, thanks

Reply:
Upgraded and tested it, everything works fine. Thanks Marco